CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Possible log types are `[add]`, `[bug]`, `[change]` and `[info]`.
```

Comment:
OK, we could also use that convention:

- `[added]` for new features.
- `[changed]` for changes in existing functionality.
- `[deprecated]` for once-stable features removed in upcoming releases.
- `[removed]` for deprecated features removed in this release.
- `[fixed]` for any bug fixes.
- `[security]` to invite users to upgrade in case of vulnerabilities.

How would you mark breaking changes? Out of these, added / changed / removed / fixed / security might be breaking.